src/plugins/home/public/application/components/home.test.tsx

```diff
@@ -230,6 +231,14 @@ describe('home', () => {
 
       expect(component.find(Welcome).exists()).toBe(false);
     });
+
+    test('should redirect to guided onboarding on Cloud instead of welcome screen', async () => {
+      defaultProps.isCloudEnabled = true;
+
+      const component = await renderHome();
+
+      expect(component.find(Welcome).exists()).toBe(false);
```

Comment:
Does it make sense to add an another `expect` statement to verify the getting started page renders?

Reply:
yes, I was not testing for the correct case here, good catch 👍